lua/telescope/builtin.lua

```diff
@@ -695,4 +695,43 @@ builtin.current_buffer_fuzzy_find = function(opts)
   }):find()
 end
 
+builtin.man_pages = function(opts)
+  opts = opts or {}
+
+  -- TODO: make man sections configurable?
+  local cmd = "apropos --sections=1 ''"
```

Comment:
drive-by: I'd definitely like to configure this. For my uses, a better default would be to search all sections.

Reply:
Can be `opts.man_cmd` or similar.